test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/util/HashMapTest.scala

```diff
@@ -16,8 +16,14 @@ import java.{util => ju}
 
 import scala.reflect.ClassTag
 
+import org.junit.Test
+import org.junit.Assert.assertNotNull
+
 class HashMapTest extends MapTest {
   def factory: HashMapFactory = new HashMapFactory
+
+  @Test def testConstructorWithLoadFactorArg(): Unit =
+    assertNotNull(new ju.HashMap[Int,String](10, 0.5f))
```

Comment:
```suggestion
    assertNotNull(new ju.HashMap[Int, String](10, 0.5f))
```
The `assertNotNull` doesn't really do anything. If the result of `new` is `null`, we have bigger problems than a bad library definition. You might as well remove it.

A comment mentioning that is this mostly to test that the constructor links would be very useful here, though.